### PR TITLE
BACKLOG-12976 : WIP - change label for single language site

### PR DESCRIPTION
**Pull request by dgriffon**
https://jira.jahia.org/browse/BACKLOG-12976

### CODE REVIEW

File: src/main/resources/javascript/locales/de.json [hunk 1]
```diff
@@ -82,7 +82,10 @@
             "clickHere": "klicken Sie hier.",
             "dialogTitle": "In Arbeit (WIP)",
             "dialogSubTitle": "Markieren Sie Ihren Content als Work in Progress wenn Sie die Veröffentlichung des Contents verhindern wollen. Um mehr über das Work in Progress Feature zu erfahren, ",
-            "label": "Als \"In Arbeit\" markieren",
+            "label" : {
+              "mark": "Als \"In Arbeit\" markieren",
+              "unmark": "Nicht als \"In Arbeit\" markieren"
```
Comment (Taybou) [inline src/main/resources/javascript/locales/de.json]: is it normal \" ?

Comment (dgriffon) [reply src/main/resources/javascript/locales/de.json]: No idea, Google provides me that translation

Comment (dgriffon) [reply src/main/resources/javascript/locales/de.json]: oh .. you were talking about the marks .. 
I used the previous one

File: src/javascript/Edit/Edit.actions.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import saveAction from './save/save.action';
 import publishAction from './publish/publish.action';
 import startWorkflow from './startWorkflow/startWorkflow.action';
 import copyLanguageAction from './copyLanguage/copyLanguage.action';
-import {Save, CloudUpload} from '@material-ui/icons';
+import {CloudUpload, Save} from '@material-ui/icons';
```
Comment (jsinovassin) [inline src/javascript/Edit/Edit.actions.js]: You can use the moonstone icons instead of material-ui

Comment (dgriffon) [reply src/javascript/Edit/Edit.actions.js]: This change is only a swap of the values (to be in alphabetical order) not to change the display of the application, please create a ticket to handle that change that cannot be part of that PR (not same scope)

**Pull request merged**